cartography/intel/aws/rds.py

```diff
@@ -98,6 +105,132 @@ def load_rds_instances(neo4j_session, data, region, current_aws_account_id, aws_
             AWS_ACCOUNT_ID=current_aws_account_id,
             aws_update_tag=aws_update_tag
         )
+        _attach_ec2_security_groups(neo4j_session, rds, aws_update_tag)
+        _attach_ec2_subnet_groups(neo4j_session, rds, aws_update_tag)
+    _attach_read_replicas(neo4j_session, read_replicas, aws_update_tag)
+
+
+def _attach_ec2_subnet_groups(neo4j_session, instance, aws_update_tag):
+    """
+    Attach RDS instance to its EC2 subnets
+    """
+    attach_rds_to_subnet_group = """
+    MERGE(sng:DBSubnetGroup{id:{DBSubnetGroupName}})
+    ON CREATE SET sng.firstseen = timestamp()
+    SET sng.vpc_id = {VpcId},
+    sng.description = {DBSubnetGroupDescription},
+    sng.status = {DBSubnetGroupStatus},
+    sng.lastupdated = {aws_update_tag}
+    WITH sng
+    MATCH(rds:RDSInstance{id:{DBInstanceArn}})
+    MERGE(rds)-[r:PART_OF_DB_SUBNET_GROUP]->(sng)
```

Comment:
Seems to deviate from our naming convention of group membership/part. Example MEMBER_OF_RDS_SUBNET_GROUP ?

Reply:
I was originally following the PART_OF_SUBNET pattern, but I can change it to the MEMBER_OF_* format. @ecdavis: thoughts?

Reply:
bit of consistency here would help. There is some consistency but not 100% there yet. Would prefer to avoid another and backtrack the previous ones once we agree on 1.

Current group nodes relationships:

(AWSGroup)<- MEMBER_AWS_GROUP
EC2SecurityGroup <- MEMBER_OF_EC2_SECURITY_GROUP
AutoScalingGroup <- MEMBER_AUTO_SCALE_GROUP
JamfComputerGroup <- MEMBER_OF_JAMF_GROUP
JamfMobileDeviceGroup <- MEMBER_OF_JAMF_GROUP